ConcatenateMatrices.java

```diff
@@ -3,37 +3,70 @@
 
 public class ConcatenateMatrices {
 
-    public static void main(String[] args) throws IOException {
-        Scanner scanner = new Scanner(new File("vstup.txt"));
-        PrintStream output = new PrintStream("vystup.txt");
-        String vstup = scanner.nextLine(); //prvy riadok (velmi pravdepodobne ide nakodit lahsie)
-        String[] rozmery = vstup.split(" ");
-        int m = Integer.parseInt(rozmery[0]);
-        int n = Integer.parseInt(rozmery[1]);
+    public static void main(String[] args){
 
-        String[][] matrix = new String[m][]; //matica a jej vynulovanie
-        for (int k = 0; k < m; k++) {
-            matrix[k] = new String[n];
+        Scanner scanner = null;
+        PrintStream output = null;
```

Comment:
Initializing variables to meaningless values is better avoided. See the comment below.